src/fu-debug.c

```diff
@@ -56,6 +66,43 @@ fu_debug_filter_cb(FuDebug *self, const gchar *log_domain, GLogLevelFlags log_le
 	return g_strv_contains((const gchar *const *)domains_str, log_domain);
 }
 
+#ifdef _WIN32
+static void
+fu_debug_handler_win32(FuDebug *self, GLogLevelFlags log_level, const gchar *msg)
+{
+	WORD type = 0x0;
+	DWORD evt_id = 0x0;
+
+	/* nothing to do */
+	if (self->event_source == NULL)
+		return;
+
+	/* map levels */
+	switch (log_level) {
+	case G_LOG_LEVEL_INFO:
+	case G_LOG_LEVEL_MESSAGE:
+		type = EVENTLOG_INFORMATION_TYPE;
+		evt_id = 0x0;
```

Comment:
Is this too noisy for windows log?

Reply:
info and message are quite unusual; I deliberately left out G_LOG_LEVEL_DEBUG